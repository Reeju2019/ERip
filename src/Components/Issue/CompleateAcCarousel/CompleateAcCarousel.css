.acSection {
  background-color: #01b8f8;
}

.caro_item{
  background-color: #01b8f8;
  height: 22vmax;
}