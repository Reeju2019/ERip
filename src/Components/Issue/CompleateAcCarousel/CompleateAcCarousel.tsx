import { Carousel, Col, Container, Row } from 'react-bootstrap'
import './CompleateAcCarousel.css'

const CompleateAcCarousel: React.FunctionComponent = () => {
  return (
    <>
      <section className='acSection'>
        <Container>
          <Carousel>
            <Carousel.Item className='caro_item'>
              <Row>
                <Col className='col-6 d-flex justify-content-center align-self-center'>
                  <h2 className=' align-self-center fs-1 text-white'>Complete AC Cleanup</h2>
                </Col>
                <Col className='col-6 d-flex justify-content-end'>
                  <img src='/static/issuePage/ac.png' alt='' />
                </Col>
              </Row>
            </Carousel.Item>
            <Carousel.Item className='caro_item'>
              <Row>
                <Col className='col-6 d-flex justify-content-center'>
                  <h2 className=' align-self-center fs-1 text-white'>Complete AC Cleanup</h2>
                </Col>
                <Col className='col-6 d-flex justify-content-end'>
                  <img src='/static/issuePage/ac.png' alt='' />
                </Col>
              </Row>
            </Carousel.Item>
            <Carousel.Item className='caro_item'>
              <Row>
                <Col className='col-6 d-flex justify-content-center'>
                  <h2 className=' align-self-center fs-1 text-white'>Complete AC Cleanup</h2>
                </Col>
                <Col className='col-6 d-flex justify-content-end'>
                  <img src='/static/issuePage/ac.png' alt='' />
                </Col>
              </Row>
            </Carousel.Item>
          </Carousel>
        </Container>
      </section>
    </>
  )
}

export default CompleateAcCarousel
