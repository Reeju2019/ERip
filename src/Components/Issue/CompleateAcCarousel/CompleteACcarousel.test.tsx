import { render } from '@testing-library/react'
import renderer from 'react-test-renderer'
import ReactDOM from 'react-dom'
import CompleateAcCarousel from './CompleateAcCarousel'



test('complete AC carousel', () => {
    render(<CompleateAcCarousel />)
  })
describe('App rendering specification', () => {
    it('SnapShot testing', () => {
      const component = renderer.create(<CompleateAcCarousel/>)
      const tree = component.toJSON()
      expect(tree).toMatchSnapshot()
    })
    it('render without crashing', () => {
      const div = document.createElement('div')
      render(<CompleateAcCarousel/>)
      ReactDOM.unmountComponentAtNode(div)
    })
  })