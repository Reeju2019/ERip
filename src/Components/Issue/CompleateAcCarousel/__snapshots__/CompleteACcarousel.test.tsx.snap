// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App rendering specification SnapShot testing 1`] = `
<section
  className="acSection"
>
  <div
    className="container"
  >
    <div
      className="carousel slide"
      onKeyDown={[Function]}
      onMouseOut={[Function]}
      onMouseOver={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
    >
      <div
        className="carousel-indicators"
      >
        <button
          aria-current={true}
          aria-label="Slide 1"
          className="active"
          data-bs-target=""
          onClick={[Function]}
          type="button"
        />
        <button
          aria-current={false}
          aria-label="Slide 2"
          data-bs-target=""
          onClick={[Function]}
          type="button"
        />
        <button
          aria-current={false}
          aria-label="Slide 3"
          data-bs-target=""
          onClick={[Function]}
          type="button"
        />
      </div>
      <div
        className="carousel-inner"
      >
        <div
          className="caro_item active carousel-item"
        >
          <div
            className="row"
          >
            <div
              className="col-6 d-flex justify-content-center align-self-center col"
            >
              <h2
                className=" align-self-center fs-1 text-white"
              >
                Complete AC Cleanup
              </h2>
            </div>
            <div
              className="col-6 d-flex justify-content-end col"
            >
              <img
                alt=""
                src="/static/issuePage/ac.png"
              />
            </div>
          </div>
        </div>
        <div
          className="caro_item carousel-item"
        >
          <div
            className="row"
          >
            <div
              className="col-6 d-flex justify-content-center col"
            >
              <h2
                className=" align-self-center fs-1 text-white"
              >
                Complete AC Cleanup
              </h2>
            </div>
            <div
              className="col-6 d-flex justify-content-end col"
            >
              <img
                alt=""
                src="/static/issuePage/ac.png"
              />
            </div>
          </div>
        </div>
        <div
          className="caro_item carousel-item"
        >
          <div
            className="row"
          >
            <div
              className="col-6 d-flex justify-content-center col"
            >
              <h2
                className=" align-self-center fs-1 text-white"
              >
                Complete AC Cleanup
              </h2>
            </div>
            <div
              className="col-6 d-flex justify-content-end col"
            >
              <img
                alt=""
                src="/static/issuePage/ac.png"
              />
            </div>
          </div>
        </div>
      </div>
      <a
        className="carousel-control-prev"
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          aria-hidden="true"
          className="carousel-control-prev-icon"
        />
        <span
          className="visually-hidden"
        >
          Previous
        </span>
      </a>
      <a
        className="carousel-control-next"
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          aria-hidden="true"
          className="carousel-control-next-icon"
        />
        <span
          className="visually-hidden"
        >
          Next
        </span>
      </a>
    </div>
  </div>
</section>
`;
